main.py

```diff
@@ -2,6 +2,41 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson"
+    ]
+    for genre in genres:
+        Genre.objects.create(
+            name=genre,
+        )
+    for actor in actors:
```

Comment:
Unpack first_name and last_name during iteration to avoid doing actor.split()[0]